mlir/lib/Dialect/Rock/Pipelines/Pipelines.cpp

```diff
@@ -48,6 +48,11 @@ void rock::buildBufferizePipeline(OpPassManager &pm,
                                   const rock::BufferizeOptions &options) {
   bool noRock = options.disableRock;
 
+  // for tosa reshape ops
+  /* rocmlir-opt --tosa-to-tensor
+   */
+  pm.addNestedPass<func::FuncOp>(tosa::createTosaToTensor());
```

Comment:
All the clone tests run this pipeline on host code, all are passing. Furthermore, tosa-to-linalg should only care about tosa ops, not the converted memref ops.

Reply:
Except there are tosa optimizations that we run or might want to run while lowering the host code that would expect a `reshape` instead of the tensor operations.

Reply:
Moving to the rock only section.